packages/copyable/src/Copyable.tsx

```diff
@@ -70,12 +70,14 @@ const codeStyle = css`
   display: inline-flex;
   align-items: center;
   height: 100%;
-  width: 100%;
+  // Button is 75px wide. Need to set a smaller width so that we're able to scroll when text overflows.
+  width: calc(100% - 75px);
   padding-left: 12px;
   border: 1px solid;
   border-radius: 4px;
   font-size: 14px;
-  overflow: hidden;
+  overflow-x: auto;
```

Comment:
I intentionally made the component non-scrollable because it doesn't look very good with a scrollbar. I think clients should just set a longer width if their content is too long. If it can't fit then maybe they should consider whether the <Code/> component better fits their use-case.

Reply:
I think you're right on this one and so I'm happy to revert the changes back around scrolling. I'll try manually setting the width if my use-case requires in mdb.design